### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aditi1403**

### CODE REVIEW

**Comment by alphaX86** [comment]
@aditi1403 you should have opened an issue regarding this! 😅😅😂. I was searching what kind it's liked to... You know the rules right?

**Comment by aditi1403** [comment]
> @aditi1403 you should have opened an issue regarding this! 😅😅😂. I was searching what kind it's liked to... You know the rules right?

Actually, I have made some grammatical edits in the readme file.

**Comment by alphaX86** [comment]
> > @aditi1403 you should have opened an issue regarding this! 😅😅😂. I was searching what kind it's liked to... You know the rules right?
> 
> Actually, I have made some grammatical edits in the readme file.

I saw that, hmm. But why without notifying in Issues? You could have said it! Please, this ain't OSS Development. There are specific rules. I'm leaving this as last. Not only you, this is for all! If anyone opens a PR without notifying they'll be considered stale and rejected, irrespective of changes you made.

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ CHATBOTS ASSISTING US :
 ![image](https://user-images.githubusercontent.com/46951797/109905711-38e70d80-7cc5-11eb-9ba7-0e891bdf21a0.png)
 
 # ABSTRACT :
-My project is mainly focusing on health-related problems.Chatbots in healthcare would be useful during this COVID-19 pandemic,the most domain NLP Platform which I've used was Dialog flow and Front end for integration.AI-powered chatbots can be used to trigger patients and guide them to receive the appropriate help. Chatbots are considered a more reliable and accurate alternative to online searches patients carry out when they’re trying to understand the cause of their symptoms.It has the capability to understand user queries.
+My project is mainly focusing on health-related problems. Chatbots in healthcare would be useful during this COVID-19 pandemic, the most domain NLP Platform which I've used was Dialog flow and Front end for integration. AI-powered chatbots can be used to trigger patients and guide them to receive the appropriate help. Chatbots are considered a more reliable and accurate alternative to online searches patients carry out when they’re trying to understand the cause of their symptoms. It can understand user queries.
```
Comment (alphaX86) [inline README.md:8]: LGTM

**Review by alphaX86 — APPROVED** [review]

**Pull request merged**